packages/core-components/api-report-alpha.md

```diff
@@ -0,0 +1,73 @@
+    readonly 'table.filter.title': 'Filters';
+    readonly 'table.filter.clearAll': 'Clear all';
+    readonly 'signIn.title': 'Sign In';
+    readonly 'signIn.loginFailed': 'Login failed';
+    readonly 'signIn.customProvider.title': 'Custom User';
+    readonly 'signIn.customProvider.subtitle2': 'This selection will not be stored.';
+    readonly 'signIn.customProvider.subtitle': 'Enter your own User ID and credentials.';
+    readonly 'signIn.customProvider.userId': 'User ID';
+    readonly 'signIn.customProvider.tokenInvalid': 'Token is not a valid OpenID Connect JWT Token';
+    readonly 'signIn.customProvider.continue': 'Continue';
+    readonly 'signIn.customProvider.idToken': 'ID Token (optional)';
+    readonly 'signIn.guestProvider.title': 'Guest';
+    readonly 'signIn.guestProvider.description': 'You will not have a verified identity, meaning some features might be unavailable.';
+    readonly 'signIn.guestProvider.enter': 'Enter';
+    readonly 'signIn.guestProvider.subtitle': 'Enter as a Guest User.';
+    readonly 'sidebar.shipToContent': 'Skip to content';
+    readonly 'sidebar.starredIntroText': 'Fun fact! As you explore all the awesome plugins in Backstage, you can actually pin them to this side nav.Keep an eye out for the little star icon (⭐) next to the plugin name and give it a click!';
+    readonly 'sidebar.recentlyViewedIntroText': 'And your recently viewed plugins will pop up here!';
+    readonly 'sidebar.dismiss': 'Dismiss';
+    readonly 'copyTextButton.tooltipText': 'Text copied to clipboard';
+    readonly 'simpleStepper.finish': 'Finish';
+    readonly 'simpleStepper.reset': 'Reset';
+    readonly 'simpleStepper.next': 'Next';
+    readonly 'simpleStepper.skip': 'Skip';
+    readonly 'simpleStepper.back': 'Back';
+    readonly 'errorPage.title': 'Looks like someone dropped the mic!';
+    readonly 'errorPage.subtitle': 'ERROR {{status}}: {{statusMessage}}';
+    readonly 'errorPage.goBack': 'Go back';
+    readonly 'errorPage.orPlease': '... or please ';
+    readonly 'errorPage.contactSupport': 'contact support';
+    readonly 'errorPage.isBug': 'if you think this is a bug.';
+    readonly 'emptyState.missingAnnotation.title': 'Missing Annotation';
+    readonly 'emptyState.missingAnnotation.actionTitle': 'Add the annotation to your component YAML as shown in the highlighted example below:';
+    readonly 'emptyState.missingAnnotation.readMore': 'Read more';
+    readonly 'emptyState.missingAnnotation.descriptionPrefix_one': 'The annotation ';
+    readonly 'emptyState.missingAnnotation.descriptionPrefix_other': 'The annotations ';
+    readonly 'emptyState.missingAnnotation.descriptionSuffix_one': ' is missing. You need to add the annotation to your component if you want to enable this tool.';
+    readonly 'emptyState.missingAnnotation.descriptionSuffix_other': ' are missing. You need to add the annotations to your component if you want to enable this tool.';
```

Comment:
Same as above, think we want to fix the interpolation to make these messages cleaner. Just leave the translations in the component as before.